libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -397,12 +397,15 @@ class MapboxNavigation(
      * Starts listening for location updates and enters an `Active Guidance` state if there's a primary route available
      * or a `Free Drive` state otherwise.
      *
+     * @param withTripService Boolean if set to false, [TripService.startService] will not be invoked and
+     * no notifications will be rendered. Default value is set to true.
      * @see [registerTripSessionStateObserver]
      * @see [registerRouteProgressObserver]
      */
     @RequiresPermission(anyOf = [ACCESS_COARSE_LOCATION, ACCESS_FINE_LOCATION])
-    fun startTripSession() {
-        tripSession.start()
+    @JvmOverloads
+    fun startTripSession(withTripService: Boolean = true) {
```

Comment:
```suggestion
    fun startTripSession(withForegroundService: Boolean = true) {
```
To make it clear what exactly is happening.